content/outcomes/application-development/prioritizing-outcomes.md

```diff
@@ -0,0 +1,170 @@
+---
+date: '2021-08-16'
+lastmod: '2021-08-16'
+layout: single
+team:
+- VMware Tanzu Labs
+title: "Prioritizing Impactful Outcomes: User Stories, Estimation, and Velocity"
+weight: 40
+---
+
+Being busy isn’t the same as making progress towards an important goal. To make progress, the team has to know what to work on, why it is important, and when one task is done and it’s time to move on to the next. The team also has to know what is the next most important and valuable thing to work on so they are always delivering the most impactful to their end users. Using user stories, estimated based on risk complexity, and ruthlessly prioritized based on that information is the best way we’ve seen to consistently and predictably deliver value forever.
+
+![Two People at a Whiteboard](/images/outcomes/application-development/whiteboard2.jpg)
+
+## What You Will Learn
+
+In this article, you will learn to:
+
+* Define what a user story is and why they are useful.
+* Describe the advantages of estimating work in terms of risk and complexity instead of time.
+* Describe how measuring velocity enables teams to predict delivery.
+
+
+## Define What a User Story is and Why They are Useful
+
+A user story is a short, simple description of a product feature told from the perspective of a user. It helps a team to understand what the feature is, who it is being built for and why it is valuable to build. 
+
+Anyone on a team can contribute stories, although it is typically the product manager’s responsibility to maintain and prioritize a healthy backlog of user stories. This includes writing and prioritizing enough user stories for a team to work on during a 1 or 2 week sprint or iteration of work. Learn more about the product manager’s role on the team in our [Balanced Teams](https://tanzu.vmware.com/developer/outcomes/application-development/balanced-teams/) article.
+
+### Why User Stories are Useful
+
+Stories are designed to** help collaboration**. Unlike weighty requirements documents, user stories capture just enough information for a team to start talking and working together. We sometimes say that a user story is a “placeholder for a conversation”. 
```

Comment:
First sentence, fix the ** between "to" and "help collaboration".

Reply:
Done